.github/workflows/dev-release.yml

```diff
@@ -0,0 +1,23 @@
+name: Development Release
+
+on:
+  push:
+    branches:
+      - canary/*
```

Comment:
So the action that was built will release to the channel specified by the branch name. It will pick up the version from the branch name as well. 

So `canary/v1.3` will release the plugin to the canary channel with the `v1.3` version. It also post-fixes the version with the current date as a base 32 string, so that way multiple v1.3 versions can be released, without having to make a new branch.

It will also pick up release notes from the pushed commits, I just threw that in as a bonus.